buddyboss/buddyboss-platform
# Fixed #897 : Activity area comment feed allowing only space

### Changes proposed in this Pull Request:



Fixes #897 

### How to test the changes in this Pull Request:

1. Login and go to the Activity page,
2. Try to comment on any activity and in the comment section just put blank spaces. 
3. Click on the "Post" button.
4. You will see the error. "Please do not leave the comment area blank."

### Proof Screenshots or Video


https://www.loom.com/share/452a190a65514eb4ada2a87be6ebfa3a
